src/rollup-plugins/rewrite-cjs-runtime-helpers.js

```diff
@@ -0,0 +1,48 @@
+import * as babel from "@babel/core";
+
+const helperPath = /^@babel\/runtime\/helpers\/esm\/(\w+)$/;
+
+export default function rewriteCjsRuntimeHelpers() {
```

Comment:
Why doesn't babel do this by default/have package.jsons that redirect?

Reply:
`package.json` redirects are not a popular technique and unfortunately they were not implemented for those, I've started a PR for this some time ago - https://github.com/babel/babel/pull/8885 but didnt get it merged.

There is a problem though - `babel` inserts requires calls to those helpers without interop helper and they just do `module.exports =`, which means their cjs shape is different than esm and unfortunately webpack@4+ doesnt try to interop such situation ( https://github.com/webpack/webpack/issues/7973 ).

So basically to make it work right now in babel we would have to introduce another set of files that would reexport those helpers as .default exports in cjs format and redirect to those, instead of regular cjs entries.

Kinda a mess, but it is what it is :/

Reply:
K, gonna merge this.